test/browser/redux/actions/async.test.js

```diff
@@ -2090,6 +2090,49 @@ describe('Asynchronous Actions', () => {
         expect(actions).to.deep.equal(expectedActions);
       });
     });
+    describe('update profile failure, unauthorized', () => {
```

Comment:
A couple, actually: https://github.com/tidepool-org/chrome-uploader/pull/317/files#diff-be6f22c962702fffabcf5837ef8f09bdR1938 and https://github.com/tidepool-org/chrome-uploader/pull/317/files#diff-be6f22c962702fffabcf5837ef8f09bdR2003